src/main/java/totemcaf/kraken/core/items/annotations/Select.java

```
@@ -11,6 +11,11 @@
 import java.util.function.Function;
 
 public interface Select {
+	/**
+	 * Uses a css selector to locate the value.
+	 * Note that not all parser types may support using css selectors
```

What does happen if the parser does not support this type of selector? Exception? Silently ignored?